modules/core/src/test/java/org/locationtech/jts/io/gml2/GMLReaderTestCase.java

```diff
@@ -11,8 +11,10 @@
  */
 package org.locationtech.jts.io.gml2;
```

Comment:
Not sure why this class is part of the PR?

Reply:
All `GMLReaderTestCase` tests always fail in IntelliJ IDEA IDE, because the files required are not present. 
Annoying but only accidental part of this PR